Rationalise link variables, and do a link review.

billy-the-fish opened the pull request.

billy-the-fish committed:
chore: update glossary internal links.

billy-the-fish committed:
Merge branch 'latest' of github.com:timescale/docs into latest

billy-the-fish committed:
chore: update (# links

billy-the-fish committed:
chore: remove duplicate markdown link variables

billy-the-fish committed:
chore: variable and link cleanup

billy-the-fish committed:
chore: cleanup

billy-the-fish committed:
Merge branch 'latest' into rationalise-link-variables

billy-the-fish committed:
Merge branch 'latest' of github.com:timescale/docs into rationalise-link-variables

# Conflicts:
#	CONTRIBUTING.md
#	_partials/_data_model_metadata.md
#	_partials/_livesync-terminal.md
#	api/administration.md
#	integrations/telegraf.md
#	mst/connection-pools.md
#	mst/ingest-data.md
#	self-hosted/install/installation-docker.md
#	self-hosted/install/installation-linux.md
#	self-hosted/install/installation-macos.md
#	self-hosted/install/installation-windows.md
#	self-hosted/migration/schema-then-data.md
#	self-hosted/multinode-timescaledb/multinode-ha.md
#	tutorials/blockchain-analyze/analyze-blockchain-query.md
#	tutorials/blockchain-query/beginner-blockchain-query.md
#	tutorials/energy-data/query-energy.md
#	tutorials/nyc-taxi-cab/query-nyc.md
#	tutorials/nyc-taxi-geospatial/plot-nyc.md
#	use-timescale/continuous-aggregates/create-a-continuous-aggregate.md
#	use-timescale/data-tiering/querying-tiered-data.md
#	use-timescale/extensions/pg-textsearch.md
#	use-timescale/hyperfunctions/function-pipelines.md
#	use-timescale/query-data/advanced-analytic-queries.md

billy-the-fish committed:
Merge branch 'latest' into rationalise-link-variables

atovpeko commented on _code-samples/toolkit/two-step_aggregation.md:
```diff
@@ -151,6 +151,8 @@ GROUP BY id, time_bucket('1 day'::interval, bucket)
 
 ### Retrospective analysis over downsampled data <a id="philosophy-retro"></a>
-[Continuous aggregates][continuous-aggregates] (or separate aggregation tables powered by a cron job or a [custom job]( __LINK__ ) ) aren't just used for speeding up queries, they're also used for [data retention]( __LINK__ ). But this can mean that they are very difficult to modify as your data ages. Unfortunately this is also when you are learning more things about the analysis you want to do o ...
+[Continuous aggregates][caggs] (or separate aggregation tables powered by a cron job or a [custom job]( __LINK__ ) ) aren't just used for speeding up queries, they're also used for [data retention]( __LINK__ ). But this can mean that they are very difficult to modify as your data ages. Unfortunately this is also when you are learning more things about the analysis you want to do on your data. By  ...
```
The LINK thing is interesting

atovpeko commented on _partials/_grafana-viz-prereqs.md:
```diff
@@ -2,9 +2,9 @@ Before you begin, make sure you have:
 
 *   Created a [Timescale][cloud-login] service.
```
```suggestion
*   Created a [$SERVICE_LONG][cloud-login].
```

atovpeko commented on _partials/_grafana-viz-prereqs.md:
```diff
@@ -2,9 +2,9 @@ Before you begin, make sure you have:
+    [Grafana Cloud][grafana-cloud].
 *   Ingested some data to your database. You can use the stock trade data from
-    the [Getting Started Guide][gsg-data].
+    the [Getting Started Guide][install].
```
I don't think this link makes any sense here

atovpeko commented on _partials/_migrate_self_postgres_timescaledb_compatibility.md:
```diff
@@ -29,4 +29,4 @@ When you build from source, best practice is to build with $PG 17.2, 16.6, etc a
 Users of [$CLOUD_LONG](https://console.cloud.timescale.com/) and platform packages for Linux, Windows, MacOS,
```
This is a missed link for console

atovpeko commented on _partials/_prereqs-cloud-project-and-self.md:
```diff
@@ -4,5 +4,5 @@ To follow the procedure on this page you need to:
    This procedure also works for [$SELF_LONG][enable-timescaledb].
 
-[create-account]: /getting-started/:currentVersion:/services/#create-a-tiger-account
+[create-account]: /getting-started/:currentVersion:/services/#create-a-service_long
```
```suggestion
[create-account]: /getting-started/:currentVersion:/services/
```

atovpeko commented on _partials/_start-using-cloud.md:
```diff
@@ -4,8 +4,7 @@ To start using $CLOUD_LONG for your data:
-
-[timescaledb]: https://www.tigerdata.com/docs/#TimescaleDB
 [connect-to-your-service]: /getting-started/:currentVersion:/services/#connect-to-your-service
+[create-a-service]: /getting-started/:currentVersion:/services/#create-a-tiger-cloud-service
```
This section is no longer called this and I think to avoid the collision between a section name and a doc name we should just link to the document in general, not a section

atovpeko commented on ai/python-interface-for-pgvector-and-timescale-vector.md:
```diff
@@ -763,3 +763,16 @@ index is only valid for one particular type of distance measure.
+[langchain]: https://python.langchain.com/docs/integrations/vectorstores/timescalevector
+[llamaindex]: https://gpt-index.readthedocs.io/en/stable/examples/vector_stores/Timescalevector.html
+[psycopg2-build-prerequisites]: https://www.psycopg.org/docs/install.html#build-prerequisites
+[sign-up-for-pgai-on-cloud_long]: https://console.cloud.timescale.com/signup?utm_campaign=vectorlaunch&utm_source=docs&utm_medium=direct
```
```suggestion
[sign-up-for-pgai-on-cloud_long]: https://console.cloud.timescale.com/signup
```

atovpeko commented on api/_hyperfunctions/time_weight/interpolated_average.md:
```diff
@@ -106,3 +106,6 @@ api_details:
 
+
+[lag]: https://www.postgresql.org/docs/current/functions-window.html#FUNCTIONS-WINDOW-TABLE
+[lead]: https://www.postgresql.org/docs/current/functions-window.html#FUNCTIONS-WINDOW-TABLE
```
these links are the same

atovpeko commented on api/_hyperfunctions/time_weight/interpolated_integral.md:
```diff
@@ -117,3 +117,6 @@ api_details:
                 user_storage_usage;
 ---
 
+
```
These links are the same

atovpeko commented on api/compression/add_compression_policy.md:
```diff
@@ -14,7 +14,7 @@ import Deprecated2180 from "versionContent/_partials/_deprecated_2_18_0.mdx";
 # add_compression_policy() <Tag type="community" content="community" />
 
-<Deprecated2180 /> Superseded by <a href="https://www.tigerdata.com/docs/api/latest/hypercore/add_columnstore_policy/">add_columnstore_policy()</a>. 
+<Deprecated2180 /> Superseded by <a href="https://www.tigerdata.com/docs/api/:currentVersion:/hypercore/add_columnstore_policy/">add_columnstore_policy()</a>. 
```
With this variable, this link does not work in the local build or instance. (Applies to this whole section)

atovpeko commented on about/changelog.md:
```diff
@@ -94,7 +94,7 @@ TimescaleDB v2.23 was released on October 29th and is now available on Tiger Clo
 ### Highlighted features in TimescaleDB v2.23.0
 
-- Simplified hypertable creation: now even easier with less configuration and smarter defaults. [Create a hypertable](https://www.tigerdata.com/docs/api/latest/hypertable/create_table/#samples) in only one step with automatic selection of all parameters. This includes:
+- Simplified hypertable creation: now even easier with less configuration and smarter defaults. [Create a hypertable](https://www.tigerdata.com/docs/api/:currentVersion:/hypertable/create_table/#samples) in only one step with automatic selection of all parameters. This includes:
```
currentVersion does not seem to work in this type of links (applies throughout this document)

atovpeko commented on api/continuous-aggregates/add_continuous_aggregate_policy.md:
```diff
@@ -83,4 +83,7 @@ Because each `batch` is an individual transaction, executing a policy in batches
+[concurrent-refresh-policies]: /use-timescale/:currentVersion:/continuous-aggregates/refresh-policies/
 [informational-views]: /api/:currentVersion:/informational-views/jobs/
+[real-time-aggregation]: /use-timescale/:currentVersion:/continuous-aggregates/real-time-aggregates/
+[utc-bucketing]: https://www.tigerdata.com/docs/use-timescale/latest/time-buckets/about-time-buckets/
```
this link looks wrong

atovpeko commented on api/data-retention/add_retention_policy.md:
```diff
@@ -50,7 +50,7 @@ call `add_retention_policy`, set `VERBOSITY verbose` to see the full context.
 |-|-|-|-|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ ...
 |`relation`|REGCLASS|-|✔| Name of the hypertable or continuous aggregate to create the policy for                                                                                                                                                                                                                                                                                                               ...
-|`drop_after`|INTERVAL or INTEGER|-|✔| Chunks fully older than this interval when the policy is run are dropped. <BR/> You specify `drop_after` differently depending on the hypertable time column type: <ul><li>TIMESTAMP, TIMESTAMPTZ, and DATE: use INTERVAL type</li><li>Integer-based timestamps: use INTEGER type. You must set <a href="https://www.tigerdata.com/docs/api/latest/hypertable/set_intege ...
+|`drop_after`|INTERVAL or INTEGER|-|✔| Chunks fully older than this interval when the policy is run are dropped. <BR/> You specify `drop_after` differently depending on the hypertable time column type: <ul><li>TIMESTAMP, TIMESTAMPTZ, and DATE: use INTERVAL type</li><li>Integer-based timestamps: use INTEGER type. You must set <a href="https://www.tigerdata.com/docs/api/:currentVersion:/hypertable/ ...
```
this link does not work

atovpeko commented on integrations/apache-airflow.md:
```diff
@@ -98,7 +98,7 @@ To exchange data between Airflow and your $SERVICE_LONG:
              you create in:
       
-             https://www.tigerdata.com/docs/getting-started/latest/try-key-features-timescale-products/#optimize-time-series-data-in-hypertables
+             https://www.tigerdata.com/docs/getting-started/:currentVersion:/try-key-features-timescale-products/#optimize-time-series-data-in-hypertables-with-hypercore
```
currentVersion does not work in those links

atovpeko commented on mst/vpc-peering/index.md:
```diff
@@ -32,7 +32,3 @@ You can set up VPC peering on:
 *   [Amazon Web Services (AWS)] [vpc-aws]
 *   [Google Cloud Platform (GCP)] [vpc-gcp]
```
These links don't work anymore.

atovpeko commented on self-hosted/migration/entire-database.md:
```diff
@@ -20,7 +20,7 @@ long time. You can continue reading from your source database during this time,
 and migrate your data from the fork. If you write to tables in your source
 database during the migration, the new writes might not be transferred to
-Timescale. To avoid this problem, see [Live migration][live-migration].
+Timescale. To avoid this problem, see [Live migration][migrate-live].
```
```suggestion
TimescaleDB. To avoid this problem, see [Live migration][migrate-live].
```

atovpeko commented on self-hosted/migration/schema-then-data.md:
```diff
@@ -32,16 +32,16 @@ retain downsampled data in a continuous aggregate for a year, the continuous
 The procedure to migrate your database requires these steps:
 
 *   [Migrate schema pre-data][migrate-schema-pre-data-link]
-*   [Restore hypertables in $TIMESCALE_DB][restore-hypertables-in-timescale-link]
```
Removing variables?

atovpeko commented on tutorials/OLD_custom-timescaledb-dashboards.md:
```diff
@@ -405,20 +405,21 @@ You can get the full code for this project from
 This tutorial was originally created for HasuraCon 2021.
 
-[![Click here to watch the video](https://assets.timescale.com/docs/images/tutorials/visualizing-compression/hasuracon-talk-thumbnail.png)](https://hasura.io/events/hasura-con-2021/talks/visualizing-timescale-db-%20compression-status-in-real-time-with-hasura/ "Watch compression status in real time with Hasura")
+[![Click here to watch the video][click-here-to-watch-the-video]](https://hasura.io/events/hasura-con-2021/talks/visualizing-timescale-db-%20compression-status-in-real-time-with-hasura/ "Watch compression status in real time with Hasura")
```
Something went wrong here. (Can we maybe get rid of those old tutorials once and for all?)

atovpeko commented on tutorials/OLD_nfl-analytics/ingest-and-query.md:
```diff
@@ -294,6 +294,7 @@ We'll use this continuous aggregate in most of the queries in the
+[caggs]: /use-timescale/:currentVersion:/continuous-aggregates/
 [extra-download]: https://assets.timescale.com/docs/downloads/nfl_2018.zip
 [kaggle-download]: https://www.kaggle.com/c/nfl-big-data-bowl-2021/data
+[toolkit_long]: https://www.tigerdata.com/docs/use-timescale/latest/install-timescaledb-toolkit/
```
```suggestion
[toolkit_long]: https://www.tigerdata.com/docs/use-timescale/:currentVersion:/install-timescaledb-toolkit/
```

atovpeko commented on use-timescale/compression/compression-on-continuous-aggregates.md:
```diff
@@ -17,11 +17,11 @@ and never modified, you can compress the aggregate to save on storage.
 <Highlight type="warning">
 Before version
-[2.18.1](https://github.com/timescale/timescaledb/releases/tag/2.18.1), you can't
+[2.18.1][2181], you can't
```
both links in this admonition block do not work

atovpeko commented on use-timescale/security/overview.md:
```diff
@@ -10,10 +10,10 @@ tags: [encryption, VPC, privacy]
 issues with $PG extensions, which strengthens the broader ecosystem as well as our own platform. $COMPANY products do not have any identified weaknesses.
 
-![Image alt](https://assets.timescale.com/docs/images/tiger-cloud-console/tiger-platform-security-overview.svg)
+![Image alt][image-alt]
```
Lol

atovpeko commented on use-timescale/services/service-overview.md:
```diff
@@ -57,6 +57,5 @@ If you need more than one database, [create a new $SERVICE_SHORT][create-service
-[pg-roles-doc]: https://www.postgresql.org/docs/current/role-attributes.html
-[create-service]: /getting-started/:currentVersion:/services/#create-a-timescale-cloud-service
-
+[create-service]: /getting-started/:currentVersion:/services/#create-a-service_long
```
```suggestion
[create-service]: /getting-started/:currentVersion:/services/
```

billy-the-fish committed:
Apply suggestions from code review

Co-authored-by: Anastasiia Tovpeko <114177030+atovpeko@users.noreply.github.com>
Signed-off-by: Iain Cox <iain@tigerdata.com>

atovpeko committed:
latest updates

atovpeko committed:
latest updates

billy-the-fish committed:
chore: Merge branch 'latest' of github.com:timescale/docs into rationalise-link-variables

# Conflicts:
#	_partials/_cloud-connect-service.md
#	_partials/_create-hypertable.md
#	_partials/_create-service.md
#	_partials/_foreign-data-wrappers.md
#	_partials/_import-data-iot.md
#	_partials/_import-data-nyc-taxis.md
#	_partials/_service-overview-azure.md
#	_partials/_service-overview.md
#	_partials/_services-intro-azure.md
#	_partials/_services-intro.md
#	getting-started/run-queries-from-console.md
#	getting-started/services.md
#	getting-started/try-key-features-timescale-products.md
#	integrations/find-connection-details.md
#	integrations/fivetran.md
#	integrations/terraform.md
#	migrate/livesync-for-s3.md
#	migrate/upload-file-using-console.md
#	migrate/upload-file-using-terminal.md
#	use-timescale/configuration/customize-configuration.md
#	use-timescale/data-tiering/enabling-data-tiering.md
#	use-timescale/query-data/about-query-data.md
#	use-timescale/query-data/select.md
#	use-timescale/security/ip-allow-list.md
#	use-timescale/security/members.md
#	use-timescale/services/service-overview.md

atovpeko commented on getting-started/run-queries-from-console.md:
```diff
@@ -72,57 +76,25 @@ To connect to a $SERVICE_SHORT:
-$SQL_ASSISTANT_SHORT in [$CONSOLE][portal-data-mode] is a chat-like interface that harnesses the power of AI to help you write, fix, and organize SQL faster and more accurately. Ask $SQL_ASSISTANT_SHORT to change existing queries, write new ones from scratch, debug error messages, optimize for query performance, add comments, improve readability—and really, get answers to any questions you can th ...
-
-<NotAvailableFreePlan />
+![$SQL_ASSISTANT](https://assets.timescale.com/docs/images/tiger-cloud-console/sql-assistant-in-action.png)
```
Need to turn this into a variable

atovpeko commented on use-timescale/configuration/customize-configuration.md:
```diff
@@ -44,7 +44,7 @@ outside of that field saves the value to be applied.
 
 <img class="main-content__illustration"
```
probably need to turn this link into a reference style one with a variable

atovpeko commented on use-timescale/services/service-overview.md:
```diff
@@ -8,54 +8,57 @@ cloud_ui:
+You use $CONSOLE_LONG to manage your $SERVICE_SHORTs and data in a convenient, centralized manner. When you [log into $CONSOLE][cloud-login], you see `Services` and other $PROJECT_SHORT management tabs: 
 
-<Tab title="Tiger Cloud on AWS" label="aws-cloud">
+![Check $SERVICE_LONG is running](https://assets.timescale.com/docs/images/tiger-on-azure/tiger-console-services-view.png)
```
should be a variable

atovpeko commented on use-timescale/services/service-overview.md:
```diff
@@ -8,54 +8,57 @@ cloud_ui:
+When you select a $SERVICE_LONG in the `Services` tab, you see `Overview` and other $SERVICE_SHORT management tabs:
 
-![Select a query to edit][select-a-query-to-edit]
+![Select a query to edit](https://assets.timescale.com/docs/images/tiger-on-azure/ops-mode-overview-tiger-console.png)
```
Should be a variable

atovpeko committed:
merge review updates

billy-the-fish committed:
chore: markdown variable links don't work in <Tags>

billy-the-fish committed:
chore: some last fixes
